src/transforms/filter.js

```diff
@@ -8,211 +8,269 @@
 
 'use strict';
 
-var isNumeric = require('fast-isnumeric');
-
-// var Lib = require('@src/lib');
 var Lib = require('../lib');
+var axisIds = require('../plots/cartesian/axis_ids');
 
-/* eslint no-unused-vars: 0*/
-
-// so that Plotly.register knows what to do with it
 exports.moduleType = 'transform';
 
-// determines to link between transform type and transform module
 exports.name = 'filter';
 
-// ... as trace attributes
 exports.attributes = {
+    active: {
```

Comment:
I am a bit bothered by `visible=true` exactly causing things to become ***in***visible.
How about `enabled`?